mbuild/formats/xyz.py

```diff
@@ -0,0 +1,48 @@
+import numpy as np
+
+import mbuild as mb
+
+__all__ = ['read_xyz']
+
+
+def read_xyz(filename):
+    """Read an XYZ file. The expected format is as follows:
+    The first line contains the number of atoms in the file The second line
+    contains a comment, which is not read.  Remaining lines, one for each
+    atom in the file, include an elemental symbol followed by X, Y, and Z
+    coordinates in Angstroms. Columns are expected tbe separated by
+    whitespace. See https://openbabel.org/wiki/XYZ_(format).
+
+    Parameters
+    ----------
+    filename : str
+        Path of the input file
+
+    Returns
+    -------
+    compound : mb.Compound
+
+    Notes
+    -----
+    The XYZ file format neglects many important details, notably as bonds,
+    residues, and box information.
+
+    There are some other flavors of the XYZ file format and not all are
+    guaranteed to be compatible with this reader. For example, the TINKER
+    XYZ format is not expected to be properly read.
+    """
+
+    compound = mb.Compound()
+
+    with open(filename, 'r') as xyz_file:
+        n_atoms = int(xyz_file.readline())
+        xyz_file.readline()
+        coords = np.zeros(shape=(n_atoms, 3), dtype=np.float64)
+        for row, _ in enumerate(coords):
+            line = xyz_file.readline().split()
+            coords[row] = line[1:4]
```

Comment:
This will raise an `IndexError` if the XYZ file contains more coordinate rows than the number of atoms defined on the first line. It might be good to have a `try` block here, so we can catch that and provide a more descriptive error message.

Reply:
We should probably also have a unit test for this scenario.

Reply:
If there are more rows than `n_atoms + 2`, wouldn't it simply stop reading the file before it reaches the end?